### ISSUE TITLE
Helm chart's kubeVersion constraint prevents deployments to AWS EKS

### ISSUE THREAD

**Issue by ozen** [opencv/cvat#7182]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)

### Steps to Reproduce

Using the latest version, when you try `helm install` to an EKS cluster, you get an error that looks like this:

```
chart requires kubeVersion: >= 1.19.0 which is incompatible with Kubernetes v1.25.13-eks-43840fb
```

### Expected Behavior

EKS with k8s version 1.25 should be satisfactory.

### Possible Solution

The reason of the error is explained in this issue: https://github.com/helm/helm/issues/10375

In summary, Helm interprets AWS distro version's `-eks` postfix as a pre-release version, and fails the version constraint. 

Since AWS won't change it's naming - at least for the existing versions - I think we should implement the workaround in the Chart.yaml file.
